torch/CMakeLists.txt

```diff
@@ -471,19 +472,36 @@ if (BUILD_TEST AND NOT NO_API AND NOT USE_ROCM)
 
   target_link_libraries(test_api torch ${TORCH_CUDA_LIBRARIES} ${CUDA_NVRTC_LIB} ${CUDA_CUDA_LIB})
 
+   #Google test of api.
+   set(TORCH_API_GTEST_DIR "${TORCH_ROOT}/test/cpp/api/")
```

Comment:
Do we need a separate variable here?

Reply:
I think I am using a separate var? It is TORCH_API_GTEST_DIR instead of TORCH_API_TEST_DIR.